### PR TITLE
Remove grouped updates feature flags

### PR DESCRIPTION
**Pull request by jurre**
This has been rolled out, the flags have been at 100% for a while now so time to clear 'em out.

### CODE REVIEW

**Review by landongrindheim — APPROVED** [review]

File: updater/spec/fixtures/bundler_grouped_by_types/updated_development_deps/Gemfile.lock [hunk 1]
```diff
@@ -32,7 +32,7 @@ GEM
     ruby-progressbar (1.13.0)
     toml-rb (2.2.0)
       citrus (~> 3.0, > 3.0)
-    unicode-display_width (2.4.2)
+    unicode-display_width (2.5.0)
```
Comment (landongrindheim) [inline updater/spec/fixtures/bundler_grouped_by_types/updated_development_deps/Gemfile.lock:35]: ❓ Is there any context around why this was bumped?

Comment (jurre) [reply updater/spec/fixtures/bundler_grouped_by_types/updated_development_deps/Gemfile.lock:35]: Did you see the commit message that bumped it? A new version was released, this is unrelated to the PR but I didn't want to go through the trouble of a separate PR to address it to geen CI green here

Comment (landongrindheim) [reply updater/spec/fixtures/bundler_grouped_by_types/updated_development_deps/Gemfile.lock:35]: Ah! I had only looked at the diff 😅 Makes sense!

**Review by jakecoffman — APPROVED** [review]

**Review by deivid-rodriguez — APPROVED** [review]
🎉

**Pull request merged**